pkg/indexmanagement/scripts.go

```diff
@@ -55,14 +66,39 @@ CMD=$(cat <<END
 from __future__ import print_function
 import json,sys
 r=json.load(sys.stdin)
-indices = [index for index in r if int(r[index]['settings']['index']['creation_date']) < $minAgeFromEpoc ]
+indices = []
+for index in r:
+  try:
+    if 'settings' in r[index]:
+      settings = r[index]['settings']
+      if 'index' in settings:
+        meta = settings['index']
+        if 'creation_date' in meta:
+          creation_date = meta['creation_date']
+          if int(creation_date) < $minAgeFromEpoc:
+            indices.append(index)
```

Comment:
as annoying as it seems, i think we also need to make sure things aren't empty...
if `r[index] and 'settings' in r[index]`

Reply:
this would mean that python is incorrectly returning key names in the for loop would it not?

Reply:
actually, i went and tested this... i was more wanting us to check against things like
`d = {'index':{'setting':{}}}` and we try something like `if 'value' in d['index']['setting']:` but the `in` alone protects us